server/src/main/java/com/cloud/server/ManagementServerImpl.java

```diff
@@ -729,6 +730,7 @@
     static final ConfigKey<Integer> vmPasswordLength = new ConfigKey<Integer>("Advanced", Integer.class, "vm.password.length", "6", "Specifies the length of a randomly generated password", false);
     static final ConfigKey<Integer> sshKeyLength = new ConfigKey<Integer>("Advanced", Integer.class, "ssh.key.length", "2048", "Specifies custom SSH key length (bit)", true, ConfigKey.Scope.Global);
     static final ConfigKey<Boolean> humanReadableSizes = new ConfigKey<Boolean>("Advanced", Boolean.class, "display.human.readable.sizes", "true", "Enables outputting human readable byte sizes to logs and usage records.", false, ConfigKey.Scope.Global);
+    static final ConfigKey<Boolean> diskProvisioningStrictness = new ConfigKey<Boolean>("Storage", Boolean.class, "disk.provisioning.type.strictness", "false", "Use storage pools only with supported disk provisioning types for disk/service offerings.", true, ConfigKey.Scope.Zone);
```

Comment:
can you please update the description here, it doesn't indicate anything related to disk provisioning strictness

Reply:
I tried to explain what provisioning strictness means. How can I make it clearer?

Reply:
@Spaceman1984 may be some sort of clear description what it is for & what it does. Something like  "_If set to true, the disk is created only when there is a suitable storage pool that supports the disk provisioning type. If set to false, the disk is created with other disk provisioning type when there are no suitable storage pool(s) that supports the original disk provisioning type. Default value is false, and this is currently supported for VMware only_."

Reply:
Done